spec/policies/hotel_policy_spec.rb

```diff
@@ -45,4 +45,34 @@
       expect(subject).to permit(user_owner)
     end
   end
+
+  permissions :set_main_image? do
+    it "User can't set main image for hotel" do
+      expect(subject).not_to permit(user)
+    end
+
+    it 'Owner can set main image for hotel' do
+      expect(subject).to permit(user_owner)
+    end
+  end
+
+  permissions :add_images? do
+    it "User can't add images for hotel" do
+      expect(subject).not_to permit(user)
+    end
+
+    it 'Owner can add images for hotel' do
+      expect(subject).to permit(user_owner)
+    end
+  end
+
+  permissions :destroy_img? do
+    it "User can't destroy hotel's images" do
+      expect(subject).not_to permit(user)
+    end
+
+    it "Owner can hotel's images" do
+      expect(subject).to permit(user_owner)
```

Comment:
just recommendation. Please read about shared_examples in rspec and create examples about owners allow and customers don't. It is a magic for create reusable tests_